framework/test/driver/BundleTest.cpp

```diff
@@ -1031,116 +1022,6 @@ void TestIllegalBundleStateChange()
   f.WaitForStop(std::chrono::seconds::zero());
 }
 
-// test failure on waiting for bundle operations.
-void TestWaitOnBundleOperation()
```

Comment:
delete the TestBundleWaitOnOperation bundle since it no longer has a purpose once this test is removed.